tests/kubeless-test-client/test-kubeless.go

```diff
@@ -41,11 +41,17 @@ func deleteNamespace(namespace string) {
 	for {
 		select {
 		case <-timeout:
+			cmd := exec.Command("kubectl", "get", "ns", namespace, "-o yaml")
```

Comment:
It will not solve our problem. Printing namespace details tell us only that namespace has "kubernetes" finalizer - but still, we don't know which object from this namespace cannot be removed. 
Could you analize "namespaced_resources_deleter.go" (https://github.com/kubernetes/kubernetes/blob/7f23a743e8c23ac6489340bbb34fa6f1d392db9d/pkg/controller/namespace/deletion/namespaced_resources_deleter.go), especially deleteAllContent method. They are using the dynamic client to detect what should be deleted. Maybe we can do the same and just print resources which are still not deleted after the timeout (6min)